### PR TITLE
Fixes a runtime in creampies

### PR DESCRIPTION
**Pull request by ghost**
```
runtime error: Cannot execute null.reaction().
proc name: splat (/obj/item/reagent_containers/food/snacks/pie/cream/proc/splat)
```

:cl: More Robust Than You
fix: fixed a creampie runtime
/:cl:

**EDIT**: wait is this a tg or hippie bug... i forgot

### CODE REVIEW

**Review by j-mie — APPROVED** [review]

File: code/modules/food_and_drinks/food/snacks_pie.dm [hunk 1]
```diff
@@ -38,7 +38,8 @@
 		return
 	var/turf/T = get_turf(hit_atom)
 	new/obj/effect/decal/cleanable/pie_smudge(T)
-	reagents.reaction(hit_atom, TOUCH)
```
Comment (ChangelingRain) [inline code/modules/food_and_drinks/food/snacks_pie.dm]: this should not be null at all.

**Comment by JamieH** [comment]
test

**Comment by MoreRobustThanYou** [comment]
gonna reopen this so Jamie can test something

**Comment by MoreRobustThanYou** [comment]
>webhook testing

**Pull request closed without merging**